appinventor/docs/reference/components/animation.html

```diff
@@ -633,13 +633,17 @@ <h3>Methods</h3>
                  <dl>
                    <dt><code>Clear()</code></dt>
                    <dd>Clears anything drawn on this Canvas but not any background color or image.</dd>
+                   <dt><code>DrawArc(number left, number top, number right, number bottom, number startAngle, number sweepAngle, boolean useCenter, boolean fill)</code></dt>
+                   <dd>Draw an arc on Canvas, by drawing a arc from a specified oval (specified by left, top, right &amp; bottom), start angle is 0 when heading to the right, and increase when rotate clockwise.</dd>
```

Comment:
Grammar: "a arc" => "an arc"

Include descriptions of useCenter and fill in this text as well.